src/main/java/core/net/OnlineWorker.java

```diff
@@ -266,6 +266,8 @@ public static boolean downloadMatchData(int matchid, MatchType matchType, boolea
 
 	public static boolean downloadMatchData( MatchKurzInfo info, boolean refresh)
 	{
+		if ( HOMainFrame.launching.get() ) return false; // do not call this on HO launch
```

Comment:
is this linked to the fix you made regarding skill of player who recently join the team? HO! tries to dtermine how many training the players received since last skill before joigning?  
also I haven't check but could we not simply avoid the function to be called during startup ?

Reply:
I completely agree with you - but I have no idea how this can be achieved. This is triggered by a change in MatchStatistic.getHomeTeamInfo, which I needed to avoid an npe. In my database I found matches with missing lineups ("Verlegenheitstruppe") that I wanted to repair by reloading.
Unfortunately, this method is also called at HO start.

Reply:
@wsbrenk ok maybe then you can add a todo explaining the issue for when a dev would have some time ... without explanation it will be quite confusing ....

Reply:
@akasolace downloadMatchData is no longer called on HO launch. I removed this line.